ui/app.js

```diff
@@ -4370,7 +4370,7 @@ $(document).ready(() => {
     } else if (eventData.type == "call") {
       const value = eventData.data;
       DispatchMAP(value);
-      if (value && value.job.includes(playerJob)) {
+      if (value && (value.job.includes(playerJob) || value.job == "FirstResponder")) {

```

Comment:
I'm fairly sure this doesn't work. `value.job` looks to me being an array, meaning that `value.job == "FirstResponder"` won't work. Same goes for the other if statement.

Reply:
I actually thought this was the case when I originally look at it as well, but during my tests I found that logging the value of `value.job` when "FirstResponder" is passed appeared to log it as a string which is why I wrote it this way.  And after making this change on my local it caused the call list to start getting populated with calls where "FirstResponder" is passed.  

I encourage you to verify this on your end if you're able to so as to ensure this is a valid solution.

Reply:
I actually thought this was the case when I originally look at it as well, but during my tests I found that logging the value of `value.job` when "FirstResponder" is passed appeared to log it as a string which is why I wrote it this way.  And after making this change on my local it caused the call list to start getting populated with calls where "FirstResponder" is passed whereas previously none were being displayed.

You do bring up a good point that I had overlooked, however.  Matching against a single string would only be true if that was the only value that was passed.  There may be cases where "FirstResponder" isn't the only piece of data passed to the job field.  I've made an update to adjust the logic to utilize the `includes()` method to check if that string exists anywhere in `value.job`.  That should avoid the aforementioned issues while still properly displaying dispatch calls in the call lists.